### PR TITLE
[#191] included Asciidoctor Kroki into the docker image

### PR DESCRIPTION
**Pull request by mskyaxl**
- included Asciidoctor-kroki(0.5.0) into the image
- updated tests and readme

### CODE REVIEW

**Comment by mojavelinux** [comment]
Seems reasonable to me.

File: README.adoc [hunk 1]
```diff
@@ -32,6 +33,8 @@ This Docker image provides:
 * Source highlighting using http://rouge.jneen.net[Rouge], https://rubygems.org/gems/coderay[CodeRay] or https://pygments.org/[Pygments]
 * https://github.com/asciidoctor/asciidoctor-confluence[Asciidoctor Confluence] {ASCIIDOCTOR_CONFLUENCE_VERSION}
 * https://github.com/asciidoctor/asciidoctor-bibtex[Asciidoctor Bibtex] {ASCIIDOCTOR_BIBTEX_VERSION}
+* https://github.com/Mogztter/asciidoctor-kroki[Asciidoctor Kroki] {ASCIIDOCTOR_KROKI_VERSION}
```
Comment (dduportal) [inline README.adoc:36]: Can you transplant this change in `README-original.adoc`?
The file `README.adoc` is generated by the command `make README` by templating `README-original.adoc` with the content of `tests/env_vars.yml`. 
This "not-easy" behavior will be fixed soon, but if you don't transplant in the PR, the next PR will lost your changes.

File: updatecli/updatecli.d/Mogztter-kroki.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    name: "Get the latest Asciidoctor-Kroki version"
+    spec:
+      owner: "Mogztter"
+      repository: "asciidoctor-kroki"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      versionFilter:
+        kind: latest
+    transformers:
+      - trimPrefix: "v"
```
Comment (dduportal) [inline updatecli/updatecli.d/Mogztter-kroki.yml:15]:
```suggestion
      versionFilter:
        kind: regex
        pattern: 'ruby-*'
    transformers:
      - trimPrefix: "ruby-v"
```
=> the "latest" will catch whataver asciidoctor-version, which happen to be the `javascript-0.14.0` while writing this comment. By specifying a pattern, you ensure that you lock on the Ruby version: did I understand correctly you intent?

File: updatecli/updatecli.d/Mogztter-kroki.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        keyword: "ARG"
+        matcher: "asciidoctor_kroki_version"
+targets:
+  updateHarnessEnvAsciidoctorKrokiVersion:
+    name: "Update the key ASCIIDOCTOR_KROKI_VERSION in the test harness env_vars.yml file"
+    kind: yaml
+    sourceID: getAsciidoctorKrokiVersion
+    spec:
+      file: "tests/env_vars.yml"
+      key: "ASCIIDOCTOR_KROKI_VERSION"
```
Comment (dduportal) [inline updatecli/updatecli.d/Mogztter-kroki.yml:32]: You need to add the line `ASCIIDOCTOR_KROKI_VERSION: 0.5.0` in the file `tests/env_vars.yml` to allow:

- Exposing the variable `ASCIIDOCTOR_KROKI_VERSION` to the test harness
- Injecting automatically the variable `ASCIIDOCTOR_KROKI_VERSION` in the generated README.adoc to ensure that it's up to date.

Sorry for this behavior, we'll simplify it soon, but it's needed.

**Review by dduportal — CHANGES_REQUESTED** [review]
Excellent job, it's almost ready to ship, thanks a lot.

There are a few minor but blocking element to update, but you got 90% of the job done 🥳

File: updatecli/updatecli.d/Mogztter-kroki.yml (hunk 3 above)
Comment (mskyaxl) [reply updatecli/updatecli.d/Mogztter-kroki.yml:32]: yeah, I realized that I missed this :) after I pushed the changes. I will update it

File: updatecli/updatecli.d/Mogztter-kroki.yml (hunk 2 above)
Comment (mskyaxl) [reply updatecli/updatecli.d/Mogztter-kroki.yml:15]: you are correct, we need the ruby version not the javascript

**Review by dduportal — APPROVED** [review]
Thanks a lot! I've was able to work on the whole README stuff. Don't worry too much on it: your change is ready to ship, and I'll take care of updating all the parameters/readme/updatecli once I've merged your PR (since I'm the one who broke the readme system earlier today :) ).

**Comment by mskyaxl** [comment]
> Thanks a lot! I've was able to work on the whole README stuff. Don't worry too much on it: your change is ready to ship, and I'll take care of updating all the parameters/readme/updatecli once I've merged your PR (since I'm the one who broke the readme system earlier today :) ).

I am still trying to look over how to run a local kroki server

**Comment by dduportal** [comment]
> I am still trying to look over how to run a local kroki server
Fair enough. If it's ok for you, I propose that we merge this PR and that you improve the test harness on a subsequent PR: does it looks good to you?

**Comment by mskyaxl** [comment]
> > I am still trying to look over how to run a local kroki server
> 
> Fair enough. If it's ok for you, I propose that we merge this PR and that you improve the test harness on a subsequent PR: does it looks good to you?

yeah, let's merge and I'll do a follow-up.

**Comment by dduportal** [comment]
For information: This PR only adds 1 Mb: 
```
asciidoctor                      latest            b3204fd6a5d0   3 hours ago    581MB
asciidoctor/docker-asciidoctor   1.12.0            cffa2ee5f6c9   4 weeks ago    580MB
```
(Don't forget it's even less in real life as these are the virtual sizes reported by the Docker Engine).

**Pull request merged**